File: Python/program/Course.py
""" Saya Apri Anggara Yudha NIM 2102585 mengerjakan soal Latihan4 dalam mata kuliah 
Desain dan Pemrograman Berorientasi Objek untuk keberkahanNya maka saya 
tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. """

class Course:
    def __init__(self, nama_matkul):
        self.__nama_matkul = nama_matkul

    # Getter
    def getNamaMatkul(self):
        return self.__nama_matkul

    # Setter
    def setNamaMatkul(self, value):
        self.__nama_matkul = value

File: CPP/program/Course.cpp
/* Saya Apri Anggara Yudha NIM 2102585 mengerjakan soal Latihan4 dalam mata kuliah 
Desain dan Pemrograman Berorientasi Objek untuk keberkahanNya maka saya 
tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. */

#include <string>

using namespace std;

class Course {

private:
    string nama_matkul;

public:
    Course () {
    }

    Course (string nama_matkul) {
        this->nama_matkul = nama_matkul;
    }

    // Getter
    string getNamaMatkul () {
        return this->nama_matkul;
    }
    
    // Setter
    void setNamaMatkul (string nama_matkul) {
        this->nama_matkul = nama_matkul;
    }
};
